.clang-format

```diff
@@ -0,0 +1,168 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+
+# AlignConsecutiveAssignments: true
```

Comment:
Let's remove all the comments, and just stick with `BasedOnStyle` and any exceptions.

Reply:
Done.